4.3/examples

```diff
@@ -0,0 +1,85 @@
+### Configuration Example
```

Comment:
Please call this file "example.md" so that I is recognised as Markdown.

Reply:
done